daemon/state.go

```diff
@@ -370,9 +371,23 @@ func (d *Daemon) regenerateRestoredEndpoints(state *endpointRestoreState) (resto
 		}(ep, epRegenerated)
 	}
 
+	var endpointCleanupCompleted sync.WaitGroup
 	for _, ep := range state.toClean {
-		go d.deleteEndpointQuiet(ep, true)
+		endpointCleanupCompleted.Add(1)
+		func() {
```

Comment:
should not be `go gunc()` here, if not the sync.WaitGroup does not make sense at all.

Reply:
🤦‍♂️

Reply:
Fixed